### PR TITLE
Added upload-artifact to snakemake action

### PR DESCRIPTION
**Pull request by mkrunic**
This PR closes issue #76 by generating some `.svg` images about the snakemake workflow.

### CODE REVIEW

File: .github/workflows/snakemake.yml
Comment (AndrewRadev) [inline .github/workflows/snakemake.yml:43]: These empty lines can be removed

File: .gitignore
Comment (AndrewRadev) [inline .gitignore:30]: You don't need to gitignore these files from the workflow, since they don't go in git. If you want to generate them for yourself locally and ignore them, that's maybe okay, but if you don't ignore them here, that's not going to make problems for the github action.

**Review by AndrewRadev — APPROVED** [review]
You should remove the binary files committed in `lib/__pycache__` and add `__pycache__` as a pattern in `.gitignore` to avoid committing them again. I should have added them myself, but I've ignored them globally, it seems.

Other than that, looks good 👍. I'll merge when you apply the fixes.

File: .gitignore
Comment (AndrewRadev) [reply .gitignore:30]: Yeah, if you want to make it easier for yourself by ignoring these, that's generally not a big deal, your call.

**Comment by mkrunic** [comment]
For some reason having two commands under one `run:` section seems to have worked now. I consolidated the two images to be generated within one step instead of two now. Should be ready to merge now!

File: .gitignore
Comment (AndrewRadev) [reply .gitignore]: Okay, considering you accidentally added the filegraph to the commit, maybe it's better to ignore them after all 😅. Instead of ignoring them in the root, however, I put them in `04_extra`.

**Pull request merged**
